packages/api/src/graphql/query.private.ts

```diff
@@ -296,35 +296,35 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
       }
     },
 
-    subscriptionAsCsv: {
+    subscriptionsAsCsv: {
       type: GraphQLString,
       args: {},
       async resolve(root, args, {dbAdapter, authenticate}) {
         const {roles} = authenticate()
         authorise(CanGetUsers, roles)
 
-        const subscriptionsList: User[] = []
+        const allSubscriptionsList: User[] = []
 
-        const getAllUsersWithSubscriptions = async ({endCursor}: any = {}) => {
+        const getAllUsersWithSubscriptions = async (after?: string | null) => {
```

Comment:
`after` is optional and can be `string` and `null`? I think that's one type to many.

Reply:
Can you explain more what you mean?